src/views/Layout.jsx

```diff
@@ -1,37 +1,19 @@
-import { Outlet, NavLink } from 'react-router-dom';
-
-import './Layout.css';
-
-/**
- * TODO: The links defined in this file don't work!
- *
- * Instead of anchor element, they should use a component
- * from `react-router-dom` to navigate to the routes
- * defined in `App.jsx`.
- */
+import { Box, Center, Flex, Heading } from '@chakra-ui/layout';
+import { Outlet, useLocation } from 'react-router-dom';
 
 export function Layout() {
+	const location = useLocation();
+
 	return (
-		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
-				</header>
-				<main className="Layout-main">
-					<Outlet />
-				</main>
-				<nav className="Nav">
-					<NavLink to="/" className="Nav-link">
-						Home
-					</NavLink>
-					<NavLink to="/list" className="Nav-link">
-						List
-					</NavLink>
-					<NavLink to="/add-item" className="Nav-link">
-						Add Item
-					</NavLink>
-				</nav>
-			</div>
-		</>
+		<Flex direction="column" p={2} h="100%">
+			<Center>
+				<Heading size={location.pathname === '/' ? '3xl' : 'xl'}>
+					List Luxe
+				</Heading>
```

Comment:
We actually added this manually to the List view, but this is smart! Will remove ours then and expect it to show. However, please add some margin to the top of the header text, compared to the wireframe there is hardly any space between the text and the upper wall of the site.

![image](https://github.com/the-collab-lab/tcl-55-smart-shopping-list/assets/99099243/f58abc39-8af6-4efe-b40f-5231bf5dc6e4)